e2e/tests/framework/exampleTemplate.e2e.spec.js

```diff
@@ -0,0 +1,124 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2022, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+/*
+* This test suite template is to be used when creating new testsuites. It will be kept up to date with the latest improvements
+* made by the Open MCT team. It will also follow our best pratices as those evolve. Please use this structure as a _reference_ and clear
+* or update any references when creating a new test suite!
+*
+* To illustrate current best practices, we've included a mocked up test suite for Renaming a Timer domain object. In this example
+* this test suite should be cloned and renamed as /e2e/tests/plugins/timer/renameTimer.e2e.spec.js
+*
+* Demonstrated:
+* - Using appActions to leverage existing functions
+* - Structure
+* - @unstable annotation
+* - await, expect, test, describe syntax
+* - Writing a custom function for a test suite
+* - Test stub for unfinished test coverage (test.fixme)
+*
+* The structure should follow
+* 1. imports
+* 2. test.describe()
+* 3. -> test1
+*    -> test2
+*    -> test3(stub)
+* 4. Any custom functions
+*
+*/
+
+//Structure: Some standard Imports. Please update the required pathing
+const { test, expect } = require('../../baseFixtures');
+const { createDomainObjectWithDefaults } = require('../../appActions');
+
+// Structure: Try to keep a single describe block per logical groups of tests. If your test runtime exceeds 5 minutes or 500 lines, it's likely that it will need to be split.
+// Annotations: Please use the @unstable tag so that our automation can pick it up as a part of our test promotion pipeline.
```

Comment:
@unlikelyzero does it make sense to default tests to unstable and mark stable ones with a "stable" flag? Right now we're asking devs/open source contributors to do this extra non-standard step that's the opposite of what was being done previously (specifying which test suites were stable).

Reply:
@khalidadil I think so, but I was trying to hit 5 birds with one stone.

We needed to tick the following boxes:
1. Provide a way to mark test and test suites as "approved for closed source execution".
2. Provide a way to mark new test and test suites as "new".
3. Provide a way to mark a test as generally flaky.
4. Mark individual tests inside of a larger test suite as unstable.
5. Get off of a string separated list inside of a npm run script.

I think we can both agree that stability, "closed source ready", and "new" tagging makes sense.

I think the pattern i'm promoting with `@unstable` assumes that we can move quickly in the open source and assume that all tests will eventually be executing without the `@unstable` tag. If we have 2000 tests (likely) it'll be easier to tag 5-10 of them as temporarily unstable.

We can always flip this in the future.